clouds/postgres/version

```diff
@@ -1 +1 @@
-1.3.0
+1.3.1
```

Comment:
Don't need to bump release only with internal chore. Please revert to 1.3.0 and we will keep the change only in the global changelog